### ISSUE TITLE
[ICC] Putricide Heroic Adds

### ISSUE THREAD

**Issue by ashido96** [dalaranwow/dalaran-wow#4749]
During the putricide transition phase on heroic both the orange and the green ooze can target the same player, they are not supposed to be able to both target the same player. The orange ooze will then run straight towards the rooted player and wipe the raid.

http://www.wowhead.com/npc=36678/professor-putricide
http://www.wowhead.com/spell=70351/unstable-experiment#used-by-npc

http://www.wowhead.com/npc=37697/volatile-ooze
http://www.wowhead.com/spell=70447/volatile-ooze-adhesive

http://www.wowhead.com/npc=37562/gas-cloud
http://www.wowhead.com/spell=70215/gaseous-bloat 

Also, reopen #4730 because @Lothloryen only fixed the comment and not the original report.

**Comment by fsironman** [comment]
Possible fix from sunwell here: https://github.com/TrinityCore/TrinityCore/issues/8680

**Comment by plznoz** [comment]
Another issue with the adds:
The Volatile Ooze should only target people with Ooze Variable on phase switch, and Gas Cloud should only target people with Gas Cloud.
Currently they can target everyone, no matter what debuff.

**Comment by ashido96** [comment]
Note: 10 man does not have the variable debufs so a check for targeting based solely on the variable (green or orange) will not be a sufficient check to eliminate this bug.

**Comment by leli66** [comment]
> Note: 10 man does not have the variable debufs so a check for targeting based solely on the variable (green or orange) will not be a sufficient check to eliminate this bug.

10man heroic mode never had variable debuffs, so thats how it should properly be, the color debuffs are only on 25heroic mode.

**Comment by Rushor** [comment]
DB/Spell: Volatile Ooze Adhesive Prevent Gaseous Bloat + Gaseous Bloat Prevent Volatile Ooze Adhesive

* updates: https://github.com/dalaranwow/dalaran-wow/issues/4749
* commit: fb6251940e5aa15b38e3e8f1f28f707611303809

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
